### DISCUSSION TITLE
Merchant Experience in the VTEX Admin

### DISCUSSION THREAD

**Discussion by beatrizmilhomem** [vtex/shoreline#555] [General]
**[🔗 Project page on Notion](https://www.notion.so/vtexhandbook/Merchant-Experience-in-the-VTEX-Admin-181f9fc14b734ddf9a7d7ead67564a34)**

### High-level objective
Find opportunities and propose improvements in the Admin experience, based on the main JTBDs. Consider that we are talking about the Admin as a platform and not about specific modules (these are still under ownership of the product teams). For example, we may propose a new information architecture for Settings but we won't propose a feature for Payment Conditions.

### Rationale for prioritization
We, as a company, don't have clarity over the main jobs executed on the Admin, how users interact with the Admin and what are their pain points. Therefore, this initiative will help to bring clarity over that.

### Scope and timeline
**Q1C1 - Product Discovery: Create base with existing knowledge**
Owner, Support: @beatrizmilhomem @EduRFonseca 

- [ ] Consolidate findings from the last survey
- [ ] Collect existing data related to the Admin experience (previous generated researchers, journeys, personas and analytics from our team or other teams)
- [ ] Define JTBDs, proto-personas and proto-journeys

___

**Q1C2 - TBD**
Owner, Support: @beatrizmilhomem @EduRFonseca 

[Do we still have any gaps that need to be filled? Can we already identify opportunities, prioritize and plan the next steps?]

**Comment by EduRFonseca** [comment]
@beatrizmilhomem Um ponto que acredito ser um objetivo macro é mapear quais são os principais JTBD, para em um futuro propor possiveis optimizações, faz sentido?

**Comment by EduRFonseca** [comment]
@beatrizmilhomem Acredito que vamos precisar criar um glossário mínimo, por exemplo, acredito que nem todos sabem o que Admin Experience contempla. Entendo que o termo é mais comum dentro do time do Admin.

**Comment by beatrizmilhomem** [comment]
Update: When we started this initiative, it came to our attention that the Research team was doing something similar. After [consolidating our initial knowledge](https://www.notion.so/vtexhandbook/Knowledge-Base-46903c17cf234d7eb241cc56c916cf7b) we found some recurrent themes. So, we decided that as the Admin team we'll focus on specific themes and work on discovering the main jobs and opportunities related to those themes.
